apps/e2e/lookup-storybook/src/app/autocomplete/autocomplete.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import {
+  SkyAutocompleteSearchAsyncArgs,
+  SkyAutocompleteSelectionChange,
+} from '@skyux/lookup';
+
+import { of } from 'rxjs';
+import { delay } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-autocomplete',
+  templateUrl: './autocomplete.component.html',
+  styleUrls: ['./autocomplete.component.scss'],
+})
+export class AutocompleteComponent {
+  public reactiveForm: FormGroup;
+
+  public templateDrivenModel: any;
+
+  public data: any[] = [
+    { name: 'Red' },
+    { name: 'Blue' },
+    { name: 'Green' },
+    { name: 'Orange' },
+    { name: 'Pink' },
+    { name: 'Purple' },
+    { name: 'Yellow' },
+    { name: 'Brown' },
+    { name: 'Turquoise' },
+    { name: 'White' },
+    { name: 'Black' },
+  ];
+
+  public templateDisabledState = false;
+
+  private reactiveDisabledState = false;
+
+  constructor(private formBuilder: FormBuilder) {
+    this.reactiveForm = this.formBuilder.group({
+      favoriteColor: undefined,
+      favoriteColorAsync: undefined,
+    });
+  }
+
+  public toggleReactiveDisabled(): void {
+    if (this.reactiveDisabledState) {
+      this.reactiveForm.get('favoriteColor')?.enable();
+    } else {
+      this.reactiveForm.get('favoriteColor')?.disable();
+    }
+
+    this.reactiveDisabledState = !this.reactiveDisabledState;
+  }
+
+  public toggleTemplateDisabled(): void {
+    this.templateDisabledState = !this.templateDisabledState;
+  }
+
+  public onSelectionChange(event: SkyAutocompleteSelectionChange): void {
+    console.log(event);
+  }
+
+  public favoriteColorSearch(args: SkyAutocompleteSearchAsyncArgs): void {
+    const searchText = (args.searchText || '').toLowerCase();
+
+    const filteredData = this.data.filter(
+      (item) => item.name.toLowerCase().indexOf(searchText) >= 0
+    );
+
+    args.result = of({
+      items: filteredData,
+      totalCount: filteredData.length,
+    }).pipe(delay(1000));
+  }
+}
```

Comment:
A lot of this code seems unnecessary, do we need anything other than the barebones for the visual test?
![Screenshot 2023-06-01 at 3 34 27 PM](https://github.com/blackbaud/skyux/assets/29147791/4d3a44c1-c7ab-4142-b061-789e1a997113)